Whiskey/Functions/Invoke-WhiskeyTask.ps1

```diff
@@ -270,6 +277,7 @@ function Invoke-WhiskeyTask
         $endedAt = Get-Date
         $duration = $endedAt - $startedAt
         Write-Verbose ('{0}  {1} in {2}' -f $prefix,$result,$duration)
+        Pop-Location

```

Comment:
This should be in a `finally` block, otherwise it won't happen if the task fails. Make sure there are assertions to check that `Pop-Location` happens when a task fails.

Reply:
It's actually in the `finally` block. It's just filled with a few other things as well, so you probably couldn't see it in the 'diff' view :)